terasoluna-gfw-common-libraries/terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraints/Compare.java

```diff
@@ -141,7 +141,7 @@
 
         /**
          * Left side object must be not equal Right side object.
-         * @since 5.3.1
+         * @since 5.3.2
          */
         NOT_EQUAL(CompareStrategy.NE),
 
```

Comment:
https://github.com/terasolunaorg/terasoluna-gfw/blob/eb357868e984e8624aa08937889c7c6c69f52618/terasoluna-gfw-common-libraries/terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraints/Compare.java#L207

- [ ] Lazy fix.

Reply:
Modified and add the previous commit.

5e531027097918bb275caa9c76547c0a356a0b85